### DISCUSSION TITLE
countdown - getHtml() save and again reloading it as component is not retaining countdown form

### DISCUSSION THREAD

**Discussion by chetan-bbc** [GrapesJS/grapesjs#6362] [Q&A] [answered]
Hi Team, please note, we are using angular 18 and node 22.9.0.
We want to use grapesjs countdown feature.

We were able to use the plugin, drag and drop worked fine. But when we save html and css and reload it again, it is failing to retain its countdown form in ui.
`data-gjs-type="countdown" `is changed to `data-gjs-type="default" `on getHtml save and reloading same.

i came across related queries and didn't found any answers for our current issue.

https://github.com/GrapesJS/components-countdown/issues/3
https://github.com/GrapesJS/components-countdown/issues/10

could you please help us with more details to try in the code or with solution to solve the issue.

Thank you for your help in advance.

**Comment by artf** [comment, marked as answer]
As mentioned already in those issues, you have to save and load back the JSON of the project, not the exported HTML/CSS/JS.

**Reply by chetan-bbc** [reply]
Thank you for replying again.
Using `const projectData = editor.getProjectData(); `and `editor.loadProjectData(projectData);` was able to resolve the count down issue.
